pyproject.toml

```diff
@@ -39,6 +39,7 @@ parallel = true
 source = ["report", "tests/unit"]
 omit = [
     "*/report/__main__.py",
+    "*/report/app.py",
```

Comment:
Like `__main__.py`, `app.py` is just one of the starter files that the cookiecutter generates and it's one that doesn't make sense to test. This project doesn't have either `__main__.py` or `app.py` but it seems unnecessary to go to the trouble of removing them from this coverage omit list